draft-ietf-rats-corim.md

```diff
@@ -303,6 +303,17 @@ convention - e.g., {{Section 4.4.1.5 of -psa-token}}).
 {::include cddl/digest.cddl}
 ~~~
 
+### Tagged Bytes Type {#sec-common-tagged-bytes}
+
+An opaque, variable-length byte string.
+It can be used in different contexts: as an instance, class or group identifier in an `environment-map`; as a raw value measurement in a `measurement-values-map`.
+Its semantics are defined by the context in which it is found, and by the overarching CoRIM profile.
+When used as an identifier the responsible allocator entity SHOULD ensure uniqueness within the usage scope.
```

Comment:
```suggestion
When used as an identifier the responsible allocator entity SHOULD ensure global uniqueness.
```

Reply:
I don't think _global_ uniqueness is needed here.  It is sufficient that the tagged bytes unambiguously identify an entity within a given scope.  The same `560(h'01')` may refer to instance "1" or to class "1" without ambiguity.

Reply:
The other examples of `instance-id` have implied global uniqueness properties.  The motivation for adding `bytes` was that some felt ueid might not be big enough to ensure global uniqueness. The verifier doesn't have additional context to know what scope to apply to disambiguate scope-A:bytes from scope-B:bytes. We shouldn't overload class properties in `instance-id` since `class-id` is intended accommodate class-based identifiers.

Reply:
Discussion today resolved that 560 is to be interpreted within the CDDL context in which it occurs. This PR proposes 3 additional contexts from the current raw-value-group context. The description of the CDDL for each context should describe the expected properties.

In the case, of `environment-map` there are other issue #176 observes the lack of meaningful scoping discussion for each of the fields (class, instance, group). If we resolve #176, then this thread becomes mute.

Reply:
> Discussion today resolved that 560 is to be interpreted within the CDDL context in which it occurs.

Yes

> This PR proposes 3 additional contexts from the current raw-value-group context.

I don't understand this.

> The description of the CDDL for each context should describe the expected properties.

Yes

> In the case, of `environment-map` there are other issue #176 observes the lack of meaningful scoping discussion for each of the fields (class, instance, group). If we resolve #176, then this thread becomes mute.

WFM